worldedit-forge/src/main/java/com/sk89q/worldedit/forge/net/handler/PacketHandlerUtil.java

```diff
@@ -42,7 +42,7 @@ static NetworkRegistry.ChannelBuilder buildLenientHandler(String id, int protoco
     private static Predicate<String> validateLenient(String protocolVersion) {
         return remoteVersion ->
             protocolVersion.equals(remoteVersion)
-                || NetworkRegistry.ABSENT.equals(remoteVersion)
+                || NetworkRegistry.ABSENT.version().equals(remoteVersion)
                 || NetworkRegistry.ACCEPTVANILLA.equals(remoteVersion);
```

Comment:
Does this one also need the same change?

Reply:
No, since the `ACCEPTVANILLA` version constant is still a `String` object, as seen here in [NetworkRegistry.java](https://github.com/MinecraftForge/MinecraftForge/blob/1.19.x/src/main/java/net/minecraftforge/network/NetworkRegistry.java#L50).